### PR TITLE
Add MFA support

### PR DESCRIPTION
**Pull request by joelfranusic-okta**
Added "MFA_ENROLL" status.
Removed "MFA_UNENROLLED", which seems to be unused or outdated?
Added documentation descriptions from developer.okta.com as IntelliSense comments on each status.

### CODE REVIEW

File: Okta.Core/Models/Embedded.cs [hunk 1]
```diff
@@ -45,5 +45,31 @@ public Embedded()
...
+
+    public class Activation : ApiObject
+    {
+        [JsonProperty("timeStep")]
+        public int TimeStep { get; set; }
+        [JsonProperty("sharedSecret")]
+        public string SharedSecret { get; set; }
+        [JsonProperty("keyLength")]
+        public int KeyLength { get; set; }
+        [JsonProperty("encoding")]
```
Comment (lboyette-okta) [inline Okta.Core/Models/Embedded.cs]: Let's add some additional white space between each of the properties

File: Okta.Core/Models/Embedded.cs [hunk 2]
```diff
@@ -45,5 +45,31 @@ public Embedded()
...
+        // Note: This is a "Factor Links Object" [0] and not a "Links Object" [1] as used elsewhere.
+        // Footnotes:
+        //   0: http://developer.okta.com/docs/api/rest/authn.html#factor-links-object
+        //   1: http://developer.okta.com/docs/api/rest/authn.html#links-object
+        [JsonProperty("_links")]
+        public Dictionary<string, FactorLink> Links { get; set; }
+    }
+    public class FactorLink : Link
+    {
+        [JsonProperty("hints")]
```
Comment (lboyette-okta) [inline Okta.Core/Models/Embedded.cs]: We should flesh out the rest of the properties of FactorLink

**Comment by joelfranusic-okta** [comment]
Updated per @lboyette-okta comments on 2015-01-23

**Comment by lboyette-okta** [comment]
lgtm

**Pull request merged**